build_docs.sh

```diff
@@ -1,11 +1,21 @@
 #!/bin/bash
+# To build manually run "sudo sh build_docs.sh"
 gem install jazzy
 echo -e "Copying MSAL public files"
-mkdir docs.temp
-mkdir docs.temp/MSAL
+mkdir -m 777 docs.temp
+mkdir -m 777 docs.temp/MSAL
 cp `find MSAL/src/public` docs.temp/MSAL
+cp `find MSAL/src/native_auth/public` docs.temp/MSAL
 cp README.md docs.temp/
-cd docs.temp
+
 echo -e "Generating MSAL documentation"
-jazzy --objc --umbrella-header MSAL/MSAL.h --framework-root . --sdk iphonesimulator --author Microsoft\ Corporation --author_url https://aka.ms/azuread  --github_url https://github.com/AzureAD/microsoft-authentication-library-for-objc --theme fullwidth
+# Generate Swift SourceKitten output
+sourcekitten doc -- -workspace MSAL.xcworkspace -scheme "MSAL (iOS Framework)" -configuration Debug RUN_CLANG_STATIC_ANALYZER=NO -sdk iphonesimulator CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO -destination 'platform=iOS Simulator,name=iPhone 14,OS=16.4' > docs.temp/swiftDoc.json
```

Comment:
are the "--" needed here?

Reply:
The lines are needed otherwise the following error happens:
```
Error: Unknown option '-x'
Usage: sourcekitten doc [--single-file] [--module-name <module-name>] [--spm] [--objc] [<arguments> ...]
  See 'sourcekitten doc --help' for more information.
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/json/common.rb:156:in `parse': 416: unexpected token at ']' (JSON::ParserError)
```